classes/Link.php

```diff
@@ -989,6 +989,22 @@ public function getMatchingUrlShopId()
      */
     public function getImageLink($name, $ids, $type = null, string $extension = 'jpg')
     {
+        $overrideUrl = Hook::exec(
+            'overrideImageLink',
+            [
+                'name' => $name,
+                'ids' => $ids,
+                'type' => $type,
+                'extension' => $extension,
+            ],
+            null,
+            true,
```

Comment:
Not compatible with PHP 7.2
```suggestion
            true
```